orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -528,6 +528,37 @@ Status OrtLoadCustomPropertyInternal(const PathString& property_folder_path,
   return Status::OK();
 }
 
+Status OrtLoadInternal(std::vector<ONNX_NAMESPACE::TensorProto>& param_tensor_protos,
+                       const PathString& checkpoint_path) {
+  // Find tensor proto files.
+  std::vector<PathString> tensor_proto_filenames;
+  FilterFilesFromDirectory(
+      checkpoint_path,
+      [&tensor_proto_filenames](const PathChar* filename) -> bool {
+        PathString filename_str = filename;
+        if (StringEndsWith(filename_str, k_tensor_proto_file_name)) {
+          tensor_proto_filenames.push_back(filename_str);
+        }
+        return true;
+      });
+
+  if (tensor_proto_filenames.empty()) {
+    return Status::OK();
+  }
```

Comment:
Is this early return necessary?

Reply:
Let's remove the check, because we loop over the `tensor_proto_filenames` and return soon after. So, the check does not add value here.